docs/sources/configure/_index.md

```diff
@@ -3624,7 +3624,19 @@ backoff_config:
   # CLI flag: -<prefix>.backoff-retries
   [max_retries: <int> | default = 10]
 
-# Enable TLS in the GRPC client. This flag needs to be enabled when any other
+# Initial stream window size. Values less than the default are not supported and
+# are ignored. Setting this to a value other than the default disables the BDP
```

Comment:
Could we define BDP?  There are 50 definitions on acronym finder.

Reply:
Hiya! I'm really not sure what the changes in the _index.md file mean sorry. I was just failing the CI `check-doc` step and the console log told me to run `make doc` and commit the changes.

Reply:
Ah gotcha.  Now I have to go try to track down which PR added these changes.

Reply:
@akhilanarayanan did you make any changes to this file? if you just `gitc checkout` this file, and also make sure you've merged the latest from `main` onto your branch, then the docs step on your PR shouldn't behave any differently than it does on `main`, and it's currently passing there.